README.md

```diff
@@ -213,6 +233,60 @@ end
 
 Each callback can be optionally configured with a set of params.  These tasks will be run in the order they are added before/after the `run!` method of the current task.  Using `halt` does not affect whether callbacks are run or not.
 
+#### Default SSH Hosts
+
+You can configure a default list of hosts to use for ssh commands made in a Task.  This will on all commands that don't specify a `:hosts` option:
```

Comment:
@jcredding forgot to address this before merging - fixed it in the hotfix commit below.  Sorry.